### ISSUE TITLE
ZCL doesn't update issues when multiple issues are referenced

### ISSUE THREAD

**Issue by Zelldon** [zeebe-io/zeebe-changelog#4]
I had a PR which references/closes multiple issues, but they are not marked with the Release label. PR https://github.com/zeebe-io/zeebe/pull/6162, issues:

> closes zeebe-io/zeebe#4002 drop multi column families usage
> closes zeebe-io/zeebe#5137 low load causes defragmentation
> zeebe-io/zeebe#5185 investigate high snapshot rate
> closes zeebe-io/zeebe#4560 unstable cluster on bigger state
> closes zeebe-io/zeebe#5810 Log file to big
> closes zeebe-io/zeebe#6038 Out of Disk: to many inodes in use
>

**Comment by Zelldon** [comment]
It seems that the extractId's can't handle text after the issue number. 
It matches the line with closes and then tries to cast it to an int, but this fails if there is more text at the end.
https://github.com/zeebe-io/zeebe-changelog/blob/f0df02a56dac5276f5e40ba3c8d2ff925e621db0/pkg/gitlog/gitlog.go#L38

**Comment by npepinpe** [comment]
I cannot reproduce this, but maybe I misunderstood how it's supposed to work. I added the following test cases in `gitlog_test.go`:

```go
		"ID with text after":            {message: "closes #4002 drop multi column families usage", issueIds: []int{4002}},
		"Multiple ID with text after":   {message: "closes #5137 low load causes defragmentation\ncloses #4560 unstable cluster on bigger state", issueIds: []int{5137, 4560}},
```

And everything passes :man_shrugging:
